### DISCUSSION TITLE
Laser Cut DXF Files download

### DISCUSSION THREAD

**Discussion by Hunter-Rohovit** [nasa-jpl/open-source-rover#367] [Q&A] [answered]
How are you supposed to download the laser cut DXF files? There isn't any "button" to download them as there is with the 3d printed STL files, and I've tried right-clicking on the raw file, and saving as a dxf file, but that only saves it as a txt file.

**Comment by sethfischer** [comment, marked as answer]
Using `arduino plate.DXF` as an example.

1. Click the "Raw" button on this page:

https://github.com/nasa-jpl/open-source-rover/blob/master/mechanical/head_assembly/laser_cut_files/arduino%20plate.DXF

2. Which will take you to:

https://raw.githubusercontent.com/nasa-jpl/open-source-rover/master/mechanical/head_assembly/laser_cut_files/arduino%20plate.DXF

Where you can Ctrl-S to save the DXF file.

Alternatively you may download the whole repo as a ZIP archive from https://github.com/nasa-jpl/open-source-rover by clicking on the "Code" button and selecting "Download ZIP" from the drop down.

![screenshot_2022-05-15_download_zip](https://user-images.githubusercontent.com/2492233/168458295-bee5d5fb-4992-4051-86df-514c1ef4f38c.png)
